### PR TITLE
[KEYCLOAK-11330] - Clustering tests for GA

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

File: .github/workflows/ci-x.yml [hunk 1]
```diff
@@ -51,3 +51,5 @@ jobs:
         run: mvn clean install -B -Pquarkus,auth-server-quarkus -DskipTests -f testsuite/pom.xml
       - name: Run base tests
         run: mvn clean install -B -Pquarkus,auth-server-quarkus -f testsuite/integration-arquillian/tests/base/pom.xml | misc/log/trimmer.sh; exit ${PIPESTATUS[0]}
+      - name: Run cluster tests
+        run: mvn clean install -B -Pauth-server-cluster-quarkus -Dsession.cache.owners=2 -Dtest=**.cluster.** -f testsuite/integration-arquillian/pom.xml  | misc/log/trimmer.sh; exit
```
Comment (stianst) [inline .github/workflows/ci-x.yml:55]: This should be a separate job

Comment (pedroigor) [reply .github/workflows/ci-x.yml:55]: Why?

Comment (pedroigor) [reply .github/workflows/ci-x.yml:76]: Maybe because you want to run them in parallel. I've changed the workflow as per your comment.

**Comment by pedroigor** [comment]
@mposolda can we merge this one?

File: .github/workflows/ci-x.yml [hunk 2]
```diff
@@ -51,3 +51,26 @@ jobs:
...
+        with:
+          name: maven-repo
+          path: .
+      - name: Extract Maven Repo
+        shell: bash
+        run: tar -xzvf maven-repo.tgz -C ~
+      - name: Build testsuite
+        run: mvn clean install -B -Pquarkus,auth-server-quarkus -DskipTests -f testsuite/pom.xml
+      - name: Run cluster tests
+        run: mvn clean install -B -Pauth-server-cluster-quarkus -Dsession.cache.owners=2 -Dtest=**.cluster.** -f testsuite/integration-arquillian/pom.xml  | misc/log/trimmer.sh; exit
```
Comment (hmlnarik) [inline .github/workflows/ci-x.yml:76]: Is this intentionally always passing due to last `exit` command that ignores exit code of `mvn`?
In the GH Actions, there is a failed test (ClientInvalidationClusterTest), but the action result is green.

Comment (pedroigor) [reply .github/workflows/ci-x.yml]: Should be fixed now.

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**